content/logs/index.md

```diff
@@ -123,10 +123,14 @@ logs:
 
 Agent 6 is able to collect logs from containers. It can be installed [on the host](https://github.com/DataDog/datadog-agent/blob/master/docs/beta/upgrade.md) or [in a container](https://github.com/DataDog/datadog-agent/tree/master/Dockerfiles/agent).
 
-For containerized installation, mount those two repositories:
-
-* `/var/run/docker.sock`: to make sure the agent can access the Docker API.
-* `/opt/datadog-agent/run`: If the agent container is restarted or removed, the registry needs to know the last logs collected for each monitored container or file.
+For containerized installation, here are the command related to log collection:
+
+* `-v /opt/datadog-agent/run:/opt/datadog-agent/run:rw`: Store on disk where to pick log file or container stdout when we restart
```

Comment:
what happens when this folder doesn't exist on the host? is it created?

Reply:
Good point, will check the behaviour in this case.

Reply:
It does indeed create the file locally